blockchain/types/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Blockchain data structures.
```

Comment:
Is there anything else going in the blockchain crate? Or just the types? I think if it's just the units going here and nothing else immediately planned, it doesn't warrant a new Blockchain folder I think. If it's only the types going in this folder in the near future - it should probably be in `transaction/blockchain/types` or `transaction/types/blockchain`

Types keep popping up I think we should chat a bit as a team about the Taxonomy of that. Like perhaps we have a types workspace/folder with the chain types?

Like perhaps we want something like a types workspace where all the types are organized? It seems like a single Blockchain folder with just a types folder creates too much fragmentation.

Reply:
FWIW I'm also adding `mc_blockchain_test_utils` in #2058.

Reply:
Oke, I voiced similar grumblings to James about this being not quite the right taxonomy, but am accepting of being outvoted in this instance 😺 - approved